### ISSUE TITLE
allow learning rate scheduler in pytorch training

### ISSUE THREAD

**Issue by sammlapp** [kitzeslab/opensoundscape#261]
in pytorch you can schedule learning rate with : 
import torch.optim.lr_scheduler.StepLR
scheduler = StepLR(optimizer, step_size=5, gamma=0.1)

in training epoch loop, you need to do `scheduler.step` 
(so we have to pass the scheduler to training function)

**Comment by sammlapp** [comment]
now possible by modifying PytorchModel's optimizer_params dictionary
